bin/cqlsh.py

```diff
@@ -478,19 +485,29 @@ def __init__(self, hostname, port, color=False,
         self.consistency_level = consistency_level
         self.serial_consistency_level = serial_consistency_level
 
+        self.secure_connect_bundle = secure_connect_bundle
+
         if use_conn:
             self.conn = use_conn
         else:
             kwargs = {}
             if protocol_version is not None:
                 kwargs['protocol_version'] = protocol_version
-            self.conn = Cluster(contact_points=(self.hostname,), port=self.port, cql_version=cqlver,
-                                auth_provider=self.auth_provider,
-                                ssl_options=sslhandling.ssl_settings(hostname, CONFIG_FILE) if ssl else None,
-                                load_balancing_policy=WhiteListRoundRobinPolicy([self.hostname]),
-                                control_connection_timeout=connect_timeout,
-                                connect_timeout=connect_timeout,
-                                **kwargs)
+            ssl_options = sslhandling.ssl_settings(hostname, CONFIG_FILE) if ssl else None
+            if secure_connect_bundle:
+                self.conn = create_cloud_cluster(secure_connect_bundle_path=self.secure_connect_bundle,
```

Comment:
I decided to make the change less intrusive than it was in the bdp code.

Instead of having one smart method to `create_cluster` with quite complex logic inside,
I have decided to explicitly create normal or cloud connection.